civihr_hrjobcontract_entities/civihr_hrjobcontract_entities.module

```diff
@@ -251,6 +251,43 @@ function civihr_hrjobcontract_entities_init() {
                     WHERE c.deleted = 0
                     ORDER BY t.job_contract_id");
 
+
+        db_query('DROP VIEW IF EXISTS hrjc_contact_details');
+        db_query("CREATE OR REPLACE VIEW hrjc_contact_details AS
+          SELECT c.id AS contact_id,
+            GROUP_CONCAT(DISTINCT CASE WHEN phone_location.name = 'Work' THEN c_phone.phone ELSE '' END
```

Comment:
hmmm not sure if this would work but isn't there is a way in drupal views to display a default value if nothing found for that column .. shouldn't it work instead of using CASE in mysql ( not sure really )